notify/__init__.py

```diff
@@ -14,10 +18,14 @@ def notify_available(clinics):
         notify_slack_available_clinics(clinics)
     if "TWITTER_CONSUMER_KEY" in os.environ:
         notify_twitter_available_clinics(clinics)
+    if "NOTIFY_CONSOLE" in os.environ:
```

Comment:
I'd like to add `and os.environ["NOTIFY_CONSOLE"].lower() in TRUE_VALUES:` to these so that if it's set to false it won't count as true. We'll need to import TRUE_VALUES

Reply:
^ just this last line also, then looks good to go

Reply:
f531111